app/src/appnav.js

```diff
@@ -177,7 +177,26 @@ const getPreloadPath = (script) => {
  */
 const openExternal = (url) => {
   log.debug(`Opening external window: ${url}`);
-  shell.openExternal(url);
+  shell.openExternal(url).then(() => {}, (reject) => {
```

Comment:
If this is a normal promise, then
```javascript
shell.openExternal(url).catch(err => log.debug(err));
```
reads cleaner. Should that log be to `warn` or `error` instead of `debug` or is that intentional?

Reply:
The error generally appears to just be `Error: Failed to open` so I don't know if a warning is appropriate. Maybe both debugs could be `info`.

Reply:
I've included an additional warning in `tryDecodeURI` because the error is just a traceback that doesn't have the input value, and updated these to `info`.